apps/storybook/src/Breadcrumbs.stories.tsx

```diff
@@ -88,6 +102,57 @@ export const Overflow: Story<BreadcrumbsProps> = (args) => {
   );
 };
 
+export const CustomOverflow: Story<BreadcrumbsProps> = (args) => {
+  const items = Array(10)
+    .fill(null)
+    .map((_, index) => (
+      <Button
+        key={index}
+        onClick={() => action(`Clicked on button ${index + 1}`)()}
+      >
+        Item {index}
+      </Button>
+    ));
+
+  return (
+    <div style={{ maxWidth: '50%', border: '1px solid lightpink', padding: 8 }}>
+      <Breadcrumbs
+        overflowButton={(visibleCount) => (
+          <DropdownMenu
+            menuItems={(close) =>
+              Array(items.length - visibleCount)
+                .fill(null)
+                .map((_, _index) => {
+                  const index = visibleCount > 1 ? _index + 1 : _index;
+                  const onClick = () => {
+                    action(`Clicked button ${index} (overflow)`)();
+                    close();
+                  };
+                  return (
+                    <MenuItem
+                      key={index}
+                      onClick={onClick}
+                      icon={<SvgPlaceholder />}
+                    >
+                      Item {index}
+                    </MenuItem>
+                  );
+                })
+            }
+          >
+            <IconButton onClick={() => action('Clicked on overflow icon')()}>
+              <SvgMore />
```

Comment:
I would say to use `SvgMoreSmall` because current one looks a bit too big for me. But just my opinion.

Reply:
![image](https://user-images.githubusercontent.com/37936169/188236946-13ba8a22-3c2a-4f7c-baa8-d323f3cc005b.png)
![image](https://user-images.githubusercontent.com/37936169/188238397-bd59e2f7-d365-4268-a59c-f97342ed48a6.png)

Buttons automatically place icons centered. I think the '...' looks better if it matches the '...' of breadcrumbs without the button such that the '...' is level with the bottom of the text if I use `SvgMoreSmall`. Is this okay with you?

Reply:
I guess it is ok.